solver/jobs.go

```diff
@@ -210,6 +211,9 @@ func (j *job) getRef(ctx context.Context, v *vertex, index Index) (Reference, er
 }
 
 func getRef(s VertexSolver, ctx context.Context, v *vertex, index Index, cache InstructionCache) (Reference, error) {
+	if v.sysMetadata.IgnoreCache {
+		logrus.Warnf("Unimplemented OpMetadata: IgnoreCache (%s, %s)", v.digest, v.name)
```

Comment:
@tonistiigi 
I'll try cache invalidation in another PR, please review the data structure?